network.go

```diff
@@ -1148,3 +1149,25 @@ func (n *network) IpamConfig() (string, []*IpamConf, []*IpamConf) {
 
 	return n.ipamType, v4L, v6L
 }
+
+func (n *network) IpamInfo() ([]*IpamInfo, []*IpamInfo) {
+	n.Lock()
```

Comment:
That is strange. Let's have this go in as is, so that you can provide the info in the docker network cmd o/p.
Then we can always change it later.